### PR TITLE
Readme: updated rant & arch diagram

### PR DESCRIPTION
**Pull request by emmanuelsearch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,38 +5,46 @@
...
+operating system for cybersecure, memory-safe, low-power IoT,
+based on Rust from the ground up, and formal verification for critical modules. 
+For a more verbose rant, see this
+[manifesto](https://future-proof-iot.github.io/RIOT-rs/dev/manifesto.html).
 
-**This is highly experimental. Expect heavy changes and breakage!**
+Overall, RIOT-rs aims for a 'batteries-included' experience, on par
+with [RIOT](https://github.com/RIOT-OS/RIOT): 
+all the needed OS facilities and a high level of integration for 
+the various libs, tools and toolchains required by low-power IoT app
```
Comment (kaspar030) [inline README.md:17]:
```suggestion
the various libs, tools and toolchains required by low-power IoT application
```

File: README.md [hunk 2]
```diff
@@ -5,38 +5,46 @@
...
+development. Targets include various hardware based on 
+32-bit microcontroller architectures (such as Arm Cortex-M, RISC-V, ESP32).
```
Comment (kaspar030) [inline README.md:19]: Drop "Arm"? and, ESP32 is not an architecture per se

Comment (emmanuelsearch) [reply README.md]: updated ;)

File: README.md (hunk 1 above)
Comment (emmanuelsearch) [reply README.md]: updated

**Comment by emmanuelsearch** [comment]
closing for https://github.com/future-proof-iot/RIOT-rs/pull/153

**Comment by jkarinkl** [comment]
https://github.com/cncf/project-template/blob/main/README-template.md this would be a nice format to go to in time.

**Pull request closed without merging**